packages/now-cli/src/util/output/create-output.ts

```diff
@@ -52,9 +52,9 @@ export default function createOutput({ debug: debugEnabled = false } = {}) {
   }
 
   function error(str: string, slug: string | null = null) {
-    log(chalk`{red.bold Error!} ${str}`, chalk.red);
+    print(`${chalk.red(`Error!`)} ${str}\n`);
     if (slug !== null) {
-      log(`More details: https://err.sh/now/${slug}`);
+      print(`More details: https://err.sh/now/${slug}\n`);
     }
   }
```

Comment:
@lucleray Why was this changed? The output stays the same, except for "Error!" not being bold, right?

Reply:
It removes the `>` in front of errors:
```
Before:
> Error! abcdef
After:
Error! abcdef
```

Reply:
Oh, I see, because `log` adds the `>`, got it